cmd/m4d/taxonomy.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mesh-for-data/mesh-for-data/pkg/taxonomy"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/cmd/helm/require"
+)
+
+func newTaxonomyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "taxonomy",
+		Short: "tools for working with taxonomies",
+		Args:  require.NoArgs,
+	}
+	cmd.AddCommand(newTaxonomySchemaCmd())
+	return cmd
+}
+
+func newTaxonomySchemaCmd() *cobra.Command {
```

Comment:
I suggest adding documentation about what this does.

Reply:
done

Reply:
Not that I mind too much in this case, but generally I disagree with the need to add these dummy docs to functions. 
Taking this one just as an example, this function is not exported, its name is descriptive, the pattern is known to most go programmers (this is how cobra works), and the logic itself of what the command does should actually be within the `Short`/`Long` arguments to the command. In any case, the code style requires function documentation to start with the function name.